### ISSUE TITLE
docpad run doesn't behave as expected

### ISSUE THREAD

**Issue by guaka** [docpad/docpad#917]
`docpad run` doesn't behave as indicated on https://docpad.org/docs/install and https://docpad.org/docs/start
It keeps asking me to agree to ToS and it doesn't create any files or run a server.

```
$ docpad run
info: Welcome to DocPad v6.70.1 (global installation: /usr/local/lib/node_modules/docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins:
info: Environment: development
Before we continue, have you read and agree to DocPad's Terms of Service (http://bevry.me/tos) and Privacy Policy (http://bevry.me/privacy)? [Y/n] Y
Fantastic. We like informed people!
$ docpad run
info: Welcome to DocPad v6.70.1 (global installation: /usr/local/lib/node_modules/docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins:
info: Environment: development
Before we continue, have you read and agree to DocPad's Terms of Service (http://bevry.me/tos) and Privacy Policy (http://bevry.me/privacy)? [Y/n] Y
Fantastic. We like informed people!
$ ls -la
total 0
drwxr-xr-x  2 guaka  staff   68 Feb 13 03:04 .
drwxr-xr-x  3 guaka  staff  102 Feb 13 03:04 ..
$
```

**Comment by guaka** [comment]
looks like https://github.com/docpad/docpad/issues/488

**Comment by guaka** [comment]
and it is!

**Comment by balupton** [comment]
Fixed in v6.71.0 February 19, 2015
